solutions/dashboard/deployment/dashboard/codacy.yaml

```diff
@@ -13,7 +13,7 @@ spec:
         spec:
           containers:
             - name: dashboard-codacy
-              image: docker.io/sanket02092001/scs:1.0.0
+              image: cortx-docker.colo.seagate.com/dashboard/codacy:1.0.0
```

Comment:
what if someone uploaded his/her images on local registry?

Reply:
This use case needs to be explored. Maybe we need to create a helm chart and create value.yaml that can be updated with custom image details.  Developer needs to update YAML files directly as of now.